resources/views/templates/user/routes.blade.php

```diff
@@ -1,7 +1,7 @@
 
 /* Auto-generated admin routes */
 Route::middleware(['auth:' . config('admin-auth.defaults.guard'), 'admin'])->group(static function () {
-    Route::prefix('admin')->namespace('Admin')->name('admin/')->group(static function() {
+    Route::prefix('admin')->namespace('App\Http\Controllers\Admin')->name('admin/')->group(static function() {
```

Comment:
Are we not going finally move admin routes to admin.php? And use custom RouteServiceProvider to use this routes?

Reply:
Good point, but let's not mix this with Laravel 8 upgrade, otherwise it's gonna be delayed even more